[WIP] Write mode polish

xtine opened the pull request:
- Excerpt toggle label just says "Show"
 - remove show more/less text toggles
- Clean up nesting of comment attachment styles
 - line up "Upload Attachment" and "Save Response" buttons

Fixes: eregs/notice-and-comment#354

<img width="628" alt="screen shot 2016-06-07 at 6 08 24 pm" src="https://cloud.githubusercontent.com/assets/24054/15879777/e133f87a-2cdb-11e6-9142-fdf4917ebb26.png">

cmc333333 commented:
Something's wrong with my local setup, so I can't test locally yet. But! Changes look good in the abstract. Is this in a state to merge?

For future reference, most of the stylesheet changes are just moving things around. New stuff:

* `.comment-context-toggle` -> `a.comment-context-toggle` (with new text color)
* `.comment-controls` has a new nested class
* `.comment-submission`
* `.comment-upload-button`
* `.comment-attachment` margin, `last-child`
* `.comment-submission`

xtine commented:
Multiple comment attachments are now stacked vertically and also accounts for extra long file names.

<img width="635" alt="screen shot 2016-06-08 at 11 42 52 am" src="https://cloud.githubusercontent.com/assets/24054/15906630/8ae61bfe-2d6e-11e6-95e8-ee1344ecc7a5.png">

xtine commented:
Added "Delete response":

<img width="634" alt="screen shot 2016-06-08 at 12 56 01 pm" src="https://cloud.githubusercontent.com/assets/24054/15908808/67835488-2d78-11e6-8a5a-4e4a839d4ae1.png">

donjo commented:
Just confirmed a few things with @jehlers:

- The actual cited text should stay black. The only part that needed to be gray (#767676) was what was in the gray bar.

- The bar should change from "Show" to "Hide" when it's expanded.

xtine commented:
- Changed the cited text back to black, that's now on eregs/notice-and-comment#361
- Toggled text now Show/Hide
- Added "Delete response" back in.

<img width="634" alt="screen shot 2016-06-08 at 12 56 01 pm" src="https://cloud.githubusercontent.com/assets/24054/15909388/05980b76-2d7b-11e6-9234-536151632b88.png">

donjo commented:
Looks good. Is the bug in eregs/notice-and-comment#367 related to this in any way or is it ready to merge?

cmc333333 commented on regulations/static/regulations/js/source/views/comment/comment-view.js:
```diff
@@ -116,6 +118,8 @@ var CommentView = Backbone.View.extend({
 
       this.$contextSectionLabel.html(options.label);
 
+      this.$deleteResponseDiv.attr('data-section', options.section);
```
This works, but we should probably switch to a template at some point

xtine commented:
The comment delete/save bug wasn't triggered with this PR.

cmc333333 commented:
General todo, we have lots and lots of `e.preventDefault();`'s now, which is concerning. I don't think we need to address that right now, however.
